components/Tabs.vue

```diff
@@ -0,0 +1,100 @@
+<script setup>
+// Tab initialisation
+const tab = ref("null");
```

Comment:
@sabiku85  During the code refactoring, I must have messed something up. At first there was a string with "description", but then it didn't work well. I looked into the Tab API in Vuetify and found that null would take care of it. "null" was probably left after the "description" string.  Null instead of the string works correctly, fixed :)